src/image-sync-manager.ts

```diff
@@ -77,7 +77,9 @@ export class ImageSyncManager {
     if (externalLayout) {
       this._onSlideViewChanged(externalLayout);
     }
-    this._imagePlayer.setActive(externalLayout === ExternalLayout.Hidden ? null : activeSlide);
+    if (activeSlide) {
```

Comment:
if we don't have active slide - dual-screen should remove child player. The changes brokes the logic